src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -267,7 +267,12 @@ public void respawn()
 		alive = true;
 	}
 
-	public void simulatePlayerMove(@NotNull Location moveLocation)
+	/**
+	 * This method moves player instantly with respect to PlayerMoveEvent
+	 *
+	 * @param moveLocation Location to move player to
+	 */
+	public void fakeMove(@NotNull Location moveLocation)
```

Comment:
New JavaDoc and method name

Reply:
👍🏻 for the docs.
But the name is still not quite right. `fakeMove` is very much misleading since that method **does** move the player, the movement is not faked since the Player is actually moved, just virtually, hence why `simulatePlayerMove(...)` would be the most appropriate and also inline with the other `simulate...()` methods naming convention.

Reply:
Ok, I will change that to simulatePlayerMove()